src/ui/components/Events/Events.tsx

```diff
@@ -33,26 +32,30 @@ function Events({ currentTime, events, executionPoint, seek }: PropsFromRedux) {
     events.map(e => e.time),
     currentTime
   );
-  return (
-    <div className="bg-bodyBgcolor py-1.5 text-xs">
-      {events.map((e, i) => {
-        return (
-          <div key={e.point}>
-            <CurrentTimeLine isActive={currentEventIndex === i} />
-            <div className="px-1.5">
-              <Event
-                onSeek={onSeek}
-                event={e}
-                currentTime={currentTime}
-                executionPoint={executionPoint}
-              />
+  if (events.length > 0) {
+    return (
+      <div className="bg-bodyBgcolor py-1.5 text-xs">
+        {events.map((e, i) => {
+          return (
+            <div key={e.point}>
+              <CurrentTimeLine isActive={currentEventIndex === i} />
+              <div className="px-1.5">
+                <Event
+                  onSeek={onSeek}
+                  event={e}
+                  currentTime={currentTime}
+                  executionPoint={executionPoint}
+                />
+              </div>
             </div>
-          </div>
-        );
-      })}
-      <CurrentTimeLine isActive={currentEventIndex === events.length && !!events.length} />
-    </div>
-  );
+          );
+        })}
+        <CurrentTimeLine isActive={currentEventIndex === events.length && !!events.length} />
+      </div>
+    );
+  } else {
+    return null;
+  }
```

Comment:
Super tiny nit that you should probably ignore– but I think we generally return null first, e.g.

```js
if (someThing) {
  return null;
}

// return actual view
```

Either way is totally fine though. This way just reads a little clearer to me because returning null at the end can be easy to miss.

Reply:
Thanks! No nit is too small for me, I appreciate you letting me know.